### PR TITLE
tests: enhance xtimer_now64_continuity

### PR DESCRIPTION
**Pull request by smlng**
- adapt code to use public, typesafe API calls
    - add some statistical output
    - minor code enhancements and doc corrections

### CODE REVIEW

File: tests/xtimer_now64_continuity/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Description
 ===========
 
-This test measures the difference of two consecutive calls to xtimer_now64() 10k times.
-Should the difference be larger then 1000us, the test fails, otherwise it succeeds.
+This test measures the difference of two consecutive calls to xtimer_now64()
+100k times. If the max or average difference is larger then 1000us the test
```
Comment (gebart) [inline tests/xtimer_now64_continuity/README.md]: "larger than"

**Comment by gebart** [comment]
Running with newlib-nano yields this output:
```
2017-03-13 09:37:24,267 - INFO # [START]
2017-03-13 09:37:24,443 - INFO # [RESULTS] min=lu, avg=lu, max=lu
2017-03-13 09:37:24,443 - INFO # [SUCCESS]
```

Do you mind changing to use the fmt module instead?
https://riot-os.org/api/group__sys__fmt.html#gae8a85f1e29de22c5f98d61350a4941e3

**Comment by smlng** [comment]
@gebart, addressed your comments, thanks!

**Comment by gebart** [comment]
ok, now I get numbers, but fmt doesn't buffer like printf does, so the output is:
```
11708[RESULTS] min=, avg=, max=
```
replacing `printf` by `print_str` yields:
```
[RESULTS] min=1, avg=1, max=708
```

See https://github.com/RIOT-OS/RIOT/pull/6544 for some background

**Comment by gebart** [comment]
https://github.com/gebart/RIOT/commit/1c36bf83b6f222acc6f13445f98bc731524f5a37 has the fix

**Comment by smlng** [comment]
@gebart thanks, is it necessary to replace `puts("[START]")` and so on by `print_str` or is just for consistency? I'd rather leave them as is.

**Comment by smlng** [comment]
@gebart merged your commit, thanks again.

**Comment by gebart** [comment]
Doesn't really matter. I simply changed them all to get rid of the stdio include. I doubt this simple test will run out of code space on any of our supported boards.

**Comment by smlng** [comment]
@gebart happy? I'll give murdock a try ...

**Pull request merged**